### DISCUSSION TITLE
[QUESTION] Blazor support roadmap

### DISCUSSION THREAD

**Discussion by kevinjpetersen** [dotnet/maui#45] [General]
It is mentioned both on the Blog Post and on the Comparison chart on this GitHub Repository, that Blazor will be supported in MAUI, however when looking at the Roadmap until release, pretty much all the other App Models are specified, except for Blazor.

Is this just an oversight, or is this something that is not gonna be there in the first GA release of MAUI?

**Comment by Phenek** [comment]
The [goals](https://github.com/dotnet/maui#goals) of MAUI

- Enable developer options to use Model-View-Update (MVU) and Blazor

First MAUI should focus on the existing platforms.
But I agree, it would be nice to see any reference of a blazor roadMap integration!

**Comment by michal-ciechan** [comment]
Once I see this in a preview version targetting Blazor, I for one (and many other people I work with), will be jumping on this 100%. At the moment, the web is such an important target, due to the ease of use (no installations, just click link and go).

In my eyes this will be the biggest draw to MAUI.

For some background, I am a WPF Dev, but recently my personal projects and slowly enterprise projects have been using Angular/React/React Native due to the importance of web!

**Comment by evolcano** [comment]
I don't understand what's "One project targeting multiple platforms and devices" really means. I wish Blazor can run directly in Native mode such Xamarin.Form , and the successor of Xamarin.Form also can run directly in browser. I don't very care about MVVM, RxUI, MVU, Blazor or something else, I just need ONE & SINGLE solution which can run in APP and Browser without much code change, and it must be Native( not script such as HTML and js).  Thanks Microsoft and you guys have break the gap between different OS, I need one more step forward to break the gap between different UI. It's means much!

**Comment by mrwcjoughin** [comment]
Watch the full video from build:
https://channel9.msdn.com/Events/Build/2020/BOD106?ocid=AID3012654&WT.mc_id=Build2020_pmmsocialblog <https://channel9.msdn.com/Events/Build/2020/BOD106?ocid=AID3012654&WT.mc_id=Build2020_pmmsocialblog>

They clearly say the intention is to build Blazor into MAUI as another platform target - so then yes it will be the UI framework to rule them all

The reason Blazor exists now is because MAUI is only officially released in .NET 6 - which is November 2021
So until then Blazor allows people to do web native better than they can now
